bigquery/api/Snippets/Test.cs

```diff
@@ -85,6 +85,15 @@ public void TestCreateDataset()
         dataset.Delete();
     }
 
+    [Fact]
+    public void TestCreateJob()
+    {
+        var snippet = new BigQueryCreateJob();
+        snippet.CreateJob(_projectId);
+        string output = _stringOut.ToString();
```

Comment:
And instead of depending on the output, we check the return of the CreateJob method, for instance, that is no null.